ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -342,6 +342,10 @@ trait CommonFtpStageSpec extends BaseSpec with Eventually {
 
       val source = listFiles("/")
 
+      eventually {
+        source.map(_.name).runWith(Sink.head).futureValue shouldBe fileName
+      }
```

Comment:
A follow-up to #2154, did you see any indications this a problem? I didn't notice an improvement for the Java tests, but still good to make sure this part is correct.

Reply:
I saw this go wrong on my machine while dealing with some permissions problems, but it's pretty likely that that doesn't have anything to do with the failures on travis. Still, good to take it out of the equation.